src/components/SimpleMesh.js

```diff
@@ -0,0 +1,23 @@
+// TODO: test it
```

Comment:
In Pixi.js V5: mesh.Mesh has move to SimpleMesh

So I renamed Mesh to SimpleMesh here.